### PR TITLE
doc: Partition Streams operator, #25468

### PR DESCRIPTION
**Pull request by patriknw**
Refs #25468

### CODE REVIEW

File: akka-docs/src/main/paradox/stream/operators/Partition.md [hunk 1]
```diff
@@ -6,11 +6,21 @@ Fan-out the stream to several streams.
 
 ## Signature
 
+@apidoc[Partition]
```
Comment (patriknw) [inline akka-docs/src/main/paradox/stream/operators/Partition.md]: I didn't find any other usage of `@signature` for graph operators, maybe this isn't too bad (and low effort)

File: akka-docs/src/main/paradox/stream/operators/Partition.md [hunk 2]
```diff
@@ -6,11 +6,21 @@ Fan-out the stream to several streams.
...
+
 ## Description
 
 Fan-out the stream to several streams. Each upstream element is emitted to one downstream consumer according to the
 partitioner function applied to the element.
 
+## Example
+
```
Comment (johanandren) [inline akka-docs/src/main/paradox/stream/operators/Partition.md]: Some friendly explanation of sample would be nice

**Review by johanandren — APPROVED** [review]
Sample is good but a little bit more text would be nice.

File: akka-docs/src/main/paradox/stream/operators/Partition.md (hunk 2 above)
Comment (patriknw) [reply akka-docs/src/main/paradox/stream/operators/Partition.md]: added

**Review by johanandren — APPROVED** [review]

**Review by chbatey — APPROVED** [review]

**Pull request merged**